2.1/nanoserver-1709/amd64/sdk/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# escape=`
+
+# Installer image
+FROM microsoft/windowsservercore:1709 AS installer-env
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# Retrieve node and git
+## https://nodejs.org/dist/v8.9.4/
+ENV NODE_VERSION 8.9.4
+ENV NODE_DOWNLOAD_SHA 48946e99ac4484e071df25741d2300f3a656f476c5ff3f8116a4746c07ebe3b7
+
+RUN Invoke-WebRequest -UseBasicParsing https://nodejs.org/dist/v${env:NODE_VERSION}/node-v${env:NODE_VERSION}-win-x64.zip -outfile node.zip; `
+    if ((Get-FileHash node.zip -Algorithm sha256).Hash -ne $env:NODE_DOWNLOAD_SHA) { `
+        Write-Host 'NODEJS CHECKSUM VERIFICATION FAILED!'; `
+        exit 1; `
+    }; `
+    `
+    Expand-Archive node.zip -DestinationPath nodejs-tmp; `
+    Move-Item nodejs-tmp/node-v${env:NODE_VERSION}-win-x64 nodejs; `
+    Remove-Item -Force node.zip; `
+    Remove-Item -Force nodejs-tmp
+
+## https://github.com/git-for-windows/git/releases
```

Comment:
I would like to know more on the motivation for adding git.  Have you been getting requests for this?  I haven't heard of any which is why I ask.  Should this be pushed to the base image?

Reply:
Now that I think about it, we can probably remove this. I believe installing it was a requirement to make bower work. We're not installing bower anymore.